### DISCUSSION TITLE
How to achieve predicted output of new energy units in Andes simulation？

### DISCUSSION THREAD

**Discussion by Crazy-Bike** [CURENT/andes#524] [Q&A]
Hello, may I ask how to achieve the given output of new energy units in Andes? For example, given the predicted output of a wind turbine, how can the wind turbine output be controlled and determined during the simulation process? I want to achieve the construction of new energy output penetration rate through this method, rather than the installed capacity penetration rate of new energy.
Thank you very much！

**Comment by jinningwang** [comment]
I assume you are talking about the fluctuated wind power generation. In ANDES documentation, following groups are designed for such functionality: RenGovernor, RenPitch, RenPlant, and RenTorque. Additionally, the following book can be a good reference for more details of these models:
1. J. H. Chow and J. J. Sanchez-Gasca, Power system modeling, computation, and control. Hoboken, NJ: Wiley-IEEE Press, 2020.

A similar frequency regulation study is given in AMS documentation, where the random load fluctuation is achieved by changing load every second: https://ltb.readthedocs.io/projects/ams/en/latest/_examples/demo/demo_AGC.html
